### DISCUSSION TITLE
Unable to reset system configuration to default values

### DISCUSSION THREAD

**Discussion by nicovoni** [SafeExamBrowser/seb-win-refactoring#1011] [Q&A]
Please, I don't understand why I can't do this, I selected this option:
`[x] Reset system configuration to default values`
and this is the output (I attach image):
![Cattura](https://github.com/user-attachments/assets/509406a5-28eb-41b7-86cc-791a49a9820d)

**Comment by dbuechel** [comment]
Please make sure that you've spelled the user name correctly and that you're actually logged-in with that particular user. If that is so, please note that there unfortunately is a minor bug in the reset utility, see #956.
